source/includes/examples/tf-example-push-logs-to-s3.rst

```diff
@@ -0,0 +1,38 @@
+.. code-block:: 
+   :copyable: true 
+
+   resource "mongodbatlas_project" "project-tf" {
```

Comment:
try to be consistent in the resource_name, `project-tf` --> `project_tf`

Reply:
Removed as this is experimental.